bom/build.gradle.kts

```diff
@@ -15,14 +15,6 @@ bomGenerator {
     includeDependency("com.firework.external.livestream", "singleHostPlayer", fireworkSdkVersion)
     // endregion
 
-    // region ExoPlayer
-    val fireworkExoPlayerVersion = "2.16.1.6"
-    includeDependency("com.firework.android.exoplayer", "exoplayer-core", fireworkExoPlayerVersion)
-    includeDependency("com.firework.android.exoplayer", "exoplayer-hls", fireworkExoPlayerVersion)
-    includeDependency("com.firework.android.exoplayer", "exoplayer-ui", fireworkExoPlayerVersion)
-    includeDependency("com.firework.android.exoplayer", "extension-ima", fireworkExoPlayerVersion)
-    // endregion
```

Comment:
Moved to Version Catalog.
For multiple reasons, one is that we do not expect any host app to depend on FwExoPlayer.